src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/IEventHubProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    // Expose to binders so they can get the EventHub connections. 
+    interface IEventHubProvider
+    {
+        EventHubClient GetSender(string eventHubName);
```

Comment:
Nit: recommend having these method names match the things they return, rather than introducing our own taxonimy